dvc/commands/data.py

```diff
@@ -114,6 +117,8 @@ def run(self) -> int:
             status = self.repo.data_status(
                 granular=self.args.granular,
                 untracked_files=self.args.untracked_files,
+                not_in_cache=True,
+                not_in_remote=self.args.not_in_remote,
```

Comment:
You might need to drop the `not_in_remote`/`not_in_cache` entries.

```python
if not self.args.not_in_remote:
	status.pop("not_in_remote")
```

Reply:
You mean for json?